docs/lake/design.md

```diff
@@ -221,8 +222,8 @@ zed lake log -p pool@branch
 ```
 To understand the log contents, the `load` operation is actually
 decomposed into two steps under the covers:
-an `add` step stores one or more
-new immutable data objects in the lake and a `commit` operation
+an "add" step stores one or more
+new immutable data objects in the lake and a "commit" operation
```

Comment:
I'm normally a fan of invoking the fixed width font in backticks to add emphasis. However, since we used to have formal `zed lake` commands called `add` and `commit`, but they're gone, I figure _not_ putting them in fixed with helps contrast with the `load` command we just mentioned that _does_ remain.

Reply:
Correct, this is described steps in the algorithm and isn't a command.